### DISCUSSION TITLE
Importing CSV

### DISCUSSION THREAD

**Discussion by Zyles** [directus/directus#16596] [Archive]
Hi,

Looking at if Directus would work for our use case.

We will create an inventory app to track inventory stock changes.

For example the table will have SKU and stock quantity field.

Now we want to import a CSV file and edit the quantity field, but we need to support two modes. Addition of quantity to current quantity, and setting quantity.

CSV file:
```
sku;quantity
1234;1
```

If import mode is "**add**" it will add +1 for SKU 1234 in quantity field = 2.

If import mode is "**set**" it will set quantity to 1 for SKU 1234 no matter what the previous value is.

Is this possible to do with the GUI or does it require heavy customization?

**Comment by br41nslug** [comment]
Doing incrementing logic like that is currently not possible in the importer without customization 😬
